### PR TITLE
WT-5013 Migrate jenkins wiredtiger perf stress job to evergreen

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

**Review by jeremythorp — APPROVED** [review]
If @alisonfel is ok with the CMake-related changes, then LGTM!

File: test/evergreen.yml [hunk 1]
```diff
@@ -3034,138 +3053,335 @@ tasks:
...
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: multi-btree-zipfian-populate
+          maxruns: 1
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: multi-btree-zipfian-workload
```
Comment (ajmorton) [inline test/evergreen.yml:3358]: Does this overwrite the `atlas_out.json` and `evergreen_out.json` from `multi-btree-zipfian-populate` above?

**Review by ajmorton — APPROVED** [review]
LGTM. One comment in evergreen.yml

File: test/evergreen.yml (hunk 1 above)
Comment (Siddhartha8899) [reply test/evergreen.yml:3358]: Yup, it overwrites `atlas_out.json` and `evergreen_out.json`, as we do not require to plot the data generated by `multi-btree-zipfian-populate`
Basically, the `multi-btree-zipfian-workload` has to run on the same directory created by `multi-btree-zipfian-populate`. Hence, it is in the same task. 
Also, in Jenkins the graph is plotted only for the read count generated from the `multi-btree-zipfian-workload`

**Review by alisonfel — APPROVED** [review]
CMake-wise (and everything else) LGTM! 👍

**Pull request merged**